packages/progress-bar/src/ProgressBar.ts

```diff
@@ -119,6 +119,9 @@ export class ProgressBar extends SizedMixin(
         if (!this.hasAttribute('role')) {
             this.setAttribute('role', 'progressbar');
         }
+        if (!this.hasAttribute('tabindex')) {
```

Comment:
This effectively sets a default tabindex of 0 when the attribute is not present. However, I have a question: what if tabindex is set to -1 initially?

In such cases, this logic will not alter the tabindex, leaving it at -1, which excludes the element from the tab order. Should we consider handling this scenario explicitly to ensure consistent keyboard accessibility?

Reply:
To create a tab stop on this from tabIndex = -1 it has to be in a roving tab index. Otherwise this progress bar will not find itself in a screen-reader!

Reply:
Why can't we skip this check `if (!this.hasAttribute('tabindex'))` and directly assign `progress-bar` a `tabIndex=0` by using just `this.setAttribute('tabindex', '0');`

Reply:
Understood!